src/python/transformation.cpp

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2021-2023 The ImpactX Community
```

Comment:
@RTSandberg Don´t forget to add this file to the respective `CMakeLists.txt` :)

Reply:
Good catch :)